Package.swift

```diff
@@ -9,23 +9,26 @@ let package = Package(
     products: [
         .library(
             name: "ZXingCppWrapper",
-            type: .static,
+            type: .dynamic,
             targets: ["ZXingCppWrapper"])
     ],
     targets: [
-        .binaryTarget(
+        .target(
             name: "ZXingCpp",
-            path: "ZXingCpp.xcframework"
+            path: "core/src",
+            publicHeadersPath: "."
         ),
         .target(
             name: "ZXingCppWrapper",
             dependencies: ["ZXingCpp"],
-            path: "Sources/Wrapper",
+            path: "wrappers/ios/Sources/Wrapper",
             publicHeadersPath: ".",
-            cxxSettings: [
-                .unsafeFlags(["-stdlib=libc++"]),
-                .unsafeFlags(["-std=gnu++17"])
+            linkerSettings: [
+                .linkedFramework("CoreGraphics"),
+                .linkedFramework("CoreImage"),
+                .linkedFramework("CoreVideo")
             ]
         )
-    ]
+    ],
+    cxxLanguageStandard: CXXLanguageStandard.gnucxx17
```

Comment:
It might be a good idea to switch to c++20 here. This will allow position-independent and multi-symbol DataMatrix detection (#344). But this comes at a performance cost, that said, my android app does not feel any different. This could follow later as well, just wanted to mention it now that I saw this line.

Reply:
Ah nice catch! According to the [documentation](https://developer.apple.com/documentation/packagedescription/cxxlanguagestandard/cxx20), that should work easily. I will still test it and feedback afterwards.